client/components/Recipes.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+// import Recipe from './Recipe'
+import { useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+
+import Card from '@mui/material/Card'
+import CardContent from '@mui/material/CardContent'
+import CardMedia from '@mui/material/CardMedia'
+import Typography from '@mui/material/Typography'
+import { Button, CardActionArea, CardActions } from '@mui/material'
+
+export default function Recipes() {
+  const recipes = useSelector((state) => state.recipes)
+
+  return (
+    <div>
+      {recipes?.map((recipe, idx) => {
+        return (
+          <>
+            <Card sx={{ maxWidth: 345, marginBottom: 2 }}>
+              <CardActionArea>
+                <CardMedia
+                  component="img"
+                  image={recipe.recipe.image}
+                  alt="recipe"
```

Comment:
make the alt tag more descriptive, you dont want multiple images with the same alt tags on the screen

Reply:
Fixed, thank you